pkg/certs/util.go

```diff
@@ -0,0 +1,55 @@
+package certs
+
+import (
+	"crypto/x509"
+	"fmt"
+	"strings"
+	"time"
+)
+
+// CertificateToString parse provided certificate to human-readable form.
+// This function should guarantee consistent output format for must-gather tooling and any code
+// that prints the certificate details.
+func CertificateToString(certificate *x509.Certificate) string {
```

Comment:
bundles are really common, I suspect you'll want a helper for that too

Reply:
yeah, tired of writing `for` loops... good idea.